aip/general/0003.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 3
+state: approved
+created: 2023-03-28
+placement:
+  category: meta
+  order: 30
+---
+
+# AIP Versioning
+
+This AIP defines the versioning scheme of the AIPs.
```

Comment:
I don't see many or any AIPs that start with "This AIP...". Please rephrase.

Reply:
https://google.aip.dev/9 is an example.

Is this guidance stated anywhere? What do you see as the disadvantage of using this form?

I'm okay with changing the text, I just think we should be clear on requirements vs trying to infer conventions.